snyk low vulnerability in lodash-merge@3.9.7

**Describe the bug**
```
simorgh git:(latest) ✗ npm run snyk               

> simorgh@0.0.1 snyk /Users/phillee/Code/BBC/simorgh
> snyk test


Testing /Users/phillee/Code/BBC/simorgh...

✗ Low severity vulnerability found in lodash.merge
  Description: Prototype Pollution
  Info: https://snyk.io/vuln/SNYK-JS-LODASHMERGE-173732
  Introduced through: assets-webpack-plugin@3.9.7
  From: assets-webpack-plugin@3.9.7 > lodash.merge@4.6.1




Organisation:      pjlee11
Package manager:   npm
Target file:       package-lock.json
Open source:       no
Project path:      /Users/phillee/Code/BBC/simorgh
Local Snyk policy: found

Tested 777 dependencies for known vulnerabilities, found 1 vulnerability, 1 vulnerable path.

Run `snyk wizard` to address these issues.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! simorgh@0.0.1 snyk: `snyk test`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the simorgh@0.0.1 snyk script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/phillee/.npm/_logs/2019-02-27T18_11_51_061Z-debug.log
```

**To Reproduce**
On `latest` branch run `npm run `snyk` see above warning. 

- [x] Initially labelled with ["bug"](https://github.com/bbc/simorgh/labels/bug)

--- a/package.json
+++ b/package.json
@@ -84,7 +84,7 @@
     "@bbc/psammead-sitewide-links": "^0.1.4",
     "@bbc/psammead-styles": "^0.2.0",
     "@bbc/psammead-visually-hidden-text": "^0.1.7",
-    "assets-webpack-plugin": "^3.9.7",
+    "assets-webpack-plugin": "^3.9.8",
     "babel-core": "^6.26.3",
     "babel-loader": "^7.1.5",
     "babel-plugin-dynamic-import-node": "^2.2.0",

